### ISSUE TITLE
tcp_routing value in install pipeline cannot be set to "disable"

### ISSUE THREAD

**Issue by wrast** [pivotal-cf/pcf-pipelines#28]
When `tcp_routing` value is set to `disable`, the `tcp_routing_ports` field is still required and configuration of the ER tile fails. See screenshot. Ports should not be required when tcp routing is disabled.

```
## TCP routing and routing services
tcp_routing: disable                   # Enable/disable TCP routing (enable|disable)
tcp_routing_ports:                   # A comma-separated list of ports and hyphen-separated port ranges, e.g. 52135,34000-35000,23478
```

![er-tile-failure](https://cloud.githubusercontent.com/assets/1258515/25487359/9cd2bb9c-2b29-11e7-9639-f3460f085adb.png)

**Comment by wrast** [comment]
Through further testing, it's apparent that the only params that can be specified to get past this issue are to enable tcp routing and also provide a valid port number. e.g.:

```
tcp_routing: enable
tcp_routing_ports: 55555
```

**Comment by ryanpei** [comment]
so @wrast this is a requirement of the tile itself, instead of the pipeline, correct?

**Comment by wrast** [comment]
@ryanpei No not in my experience. The tile config allows tcp routing to be disabled. This is the relevant portion in the tile's UI (located at bottom of Networking section):

<img width="1009" alt="screen shot 2017-05-01 at 1 04 14 pm" src="https://cloud.githubusercontent.com/assets/1258515/25590683/c76463fe-2e6e-11e7-9459-9ea8faf4d18d.png">

Clicking to enable it then allows you to enter port(s). I was unable to set `tcp_routing` simply to `disable` in the params file.

**Comment by ryanpei** [comment]
thanks @wrast , so this may be an issue with Ops Mgr API endpoint for configuration... I'll check with @ryanmoran and @sadvani to see if it should be possible to configure a selector without also configuring the field for an option that isn't selected in this particular configuration.

**Comment by ryanmoran** [comment]
@ryanpei @wrast The correct behavior when configuring the ERT is to set the `.properties.tcp_routing` to `disable`, and to omit the `.properties.tcp_routing.enable.reservable_ports`. The ERT fully supports that configuration. If there is an issue with what you are seeing here, its in this tooling, and not in ERT or OpsMan.

**Comment by ryanpei** [comment]
thanks @ryanmoran . we'll look at fixing this on our pipeline then.

**Comment by krishicks** [comment]
More logs from @forddonald in #50.

**Comment by ryanpei** [comment]
this is now fixed as of v0.13.1
